Advanced-Solidity.md

```diff
@@ -130,12 +130,12 @@ contract Enum {
         return status;
     }
 
-    // Pass a uint for input to update the value
+    // Pass a Status enum value to update the status
     function set(Status _status) public {
         status = _status;
     }
 
-    // Update value to a specific enum members
+    // Set status to the Canceled enum value
```

Comment:
This is an example of updating a value to a specific enum member, but yes, it is updating to the Canceled value. Maybe you can the comment as such - 

`// Update status enum value to a specific enum member, in this case, to the Canceled enum value`

Reply:
I appreciate your suggestion. 
I have updated the comments to reflect this and modified the wording to better suit both intentions.